GenericCamera/v2/zwo_2x2.yaml

```diff
@@ -2,7 +2,7 @@ instances:
   - sal_index: 1
     ip: 139.229.170.216
     directory: ~/data/
-    camera: Zwo
+    camera: zwo
     config:
-      binValue: 2
-      useZWOFilterWheel: True
+      bin_value: 2
+      use_zwo_filter_wheel: True
```

Comment:
Use true/false, not True/False in yaml (and json) files. Please look for and fix all instances.

Reply:
Done.